packages/spark/base/inputs/_text-input.scss

```diff
@@ -20,6 +20,162 @@
   border-color: $sprk-text-input-focus-border-color;
 }
 
+// Applies to any huge input
+.sprk-b-InputContainer--huge .sprk-b-TextInput {
+  position: relative;
```

Comment:
We will want to make sure that the css for the huge inputs doesn't get put back into this file. Amber moved it all to the input_container file in a recent merge.